pages/[chain]/[address].tsx

```diff
@@ -400,7 +406,8 @@ const SynthPage: React.FC<Props> = ({ data, chainId, relatedSynths }) => {
                 </ul>
               </div>
             </>
-          ) : (
+          )}
+          {data.type === "lsp" && showLspForm && (
```

Comment:
even if `data.type` assertion is part of the `showLspForm`, I had to keep the condition here to avoid some build errors. Props.data is an union type and `<LSP />` components expects only LSP synths